.github/workflows/ci.yaml

```diff
@@ -50,20 +54,23 @@ jobs:
           bundler-cache: true
         timeout-minutes: 30
       - name: Run SQLite tests
+        continue-on-error: ${{ matrix.rails-version != '6.1' }}
```

Comment:
Alternative, we can manager the "allowed to fail" combinations by creating a new matrix entry called "allow-failure" and leverage the "include" to set those for specific combos

Reply:
I like the matrix having "bail on this version".

That is very clean